index.html

```diff
@@ -121,9 +123,8 @@
     <td>
       <ul>
         <li>Above
-        <li><a href="https://github.com/tc39/test262">Test262</a> acceptance tests have been written for mainline usage scenarios, and merged
-        <li>Two compatible implementations which pass the acceptance tests
-        <li>Significant in-the-field experience with shipping implementations, such as that provided by two independent VMs
+        <li>Test262 tests are merged
+        <li>Two complete browser implementations, shipped in a preview release, which pass the test262 tests
```

Comment:
Enfranchising browsers here does have the effect of *dis*enfranchising non-browser implentations. I suspect that this will be controversial.